cabal.project

```diff
@@ -39,11 +39,15 @@ package lsm-tree
   -- relevant mostly only for development & testing
   ghc-options: -fno-ignore-asserts
 
+  -- TODO: temporarily disable asynchronous HasBlockIO on Linux-based systems.
+  -- See lsm-tree#129.
+  flags: +serialblockio
```

Comment:
this flag doesn't need to be enabled for all, and especially not for CI.

Currently it's hiding some bug somewhere.

Reply:
I'm looking into the bug, but I didn't want to have that hold back other PRs